Userland/Utilities/man.cpp

```diff
@@ -99,4 +141,8 @@ int main(int argc, char* argv[])
 
     String rendered = document->render_for_terminal(view_width);
     out("{}", rendered);
+
+    fclose(stdout);
+    int wstatus;
+    waitpid(pager_pid, &wstatus, 0);
```

Comment:
what if someone `^Z`'s the pager?

Reply:
Couldn't `man` just `disown` the child and exit? Or does that not work because of `stdout` and friends?

Reply:
Hmm...
I'm not sure how that would play out with stdio's buffering and the fact that `less` doesn't read everything unless it has to...

Reply:
I don't think the wait should be necessary, but it is for some reason. I don't know enough about job control to fix it.

Reply:
Is this necessary to get this merged, or is it something I can fix later?

Reply:
:shrug: just slap a FIXME on it and do it later :P